### PR TITLE
AddControllers

### PR DESCRIPTION
**Pull request by kozeyandrey**

### CODE REVIEW

**Comment by monobook** [comment]
Fos_rest section will be more huge in future, I guess it would be nice to move it to separate yml file.

File: src/AppBundle/Controller/EquipmentContributeController.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+     * }
+     * )
+     *
+     *
+     * RestView()
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getEquipmentContributeAction()
```
Comment (spolischook) [inline src/AppBundle/Controller/EquipmentContributeController.php]: getEquipmentContributesAction

File: src/AppBundle/Controller/EquipmentResourceController.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     * }
+     * )
+     *
+     *
+     * RestView()
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getEquipmentResourceAction()
```
Comment (spolischook) [inline src/AppBundle/Controller/EquipmentResourceController.php]: getEquipmentResourcesAction

File: src/AppBundle/Controller/ExportedUserController.php [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+namespace AppBundle\Controller;
+
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use FOS\RestBundle\Controller\FOSRestController;
+use Nelmio\ApiDocBundle\Annotation\ApiDoc;
+use FOS\RestBundle\Controller\Annotations\View as RestView;
+use FOS\RestBundle\View\View;
+
+class ExportedUserController extends FOSRestController
```
Comment (spolischook) [inline src/AppBundle/Controller/ExportedUserController.php]: remove this

File: src/AppBundle/Controller/FaqController.php [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+     * }
+     * )
+     *
+     *
+     * RestView()
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getFaqAction()
```
Comment (spolischook) [inline src/AppBundle/Controller/FaqController.php]: getFaqsAction

File: src/AppBundle/Controller/FinancialContributeController.php [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+     * }
+     * )
+     *
+     *
+     * RestView()
+     * @return mixed
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getFinancialContributeAction()
```
Comment (spolischook) [inline src/AppBundle/Controller/FinancialContributeController.php]: getFinancialContributesAction

File: src/AppBundle/Controller/FinancialResourceController.php [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+     * )
+     *
+     *
+     * RestView()
+     * @param
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getFinancialResourceAction()
```
Comment (spolischook) [inline src/AppBundle/Controller/FinancialResourceController.php]: getFinancialResourcesAction

File: src/AppBundle/Controller/OtherContributeController.php [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+     * }
+     * )
+     *
+     * RestView()
+     * @param $title
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getOtherContributeAction($title)
```
Comment (spolischook) [inline src/AppBundle/Controller/OtherContributeController.php]: search by id or slug, not by title

File: src/AppBundle/Controller/StatusController.php [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+     * }
+     * )
+     *
+     * RestView()
+     * @param $title
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getStatusAction($title)
```
Comment (spolischook) [inline src/AppBundle/Controller/StatusController.php]: search by slug, not by title

File: src/AppBundle/Controller/UserController.php [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+     * }
+     * )
+     *
+     * RestView()
+     * @param $firstName
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getUserAction($firstName)
```
Comment (spolischook) [inline src/AppBundle/Controller/UserController.php]: really? what about two users with same firstName?

File: src/AppBundle/Controller/WorkContributeController.php [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+     * )
+     *
+     *
+     * RestView()
+     * @param
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getWorkContributeAction()
```
Comment (spolischook) [inline src/AppBundle/Controller/WorkContributeController.php]: getWorkContributesAction

File: src/AppBundle/Controller/WorkResourceController.php [hunk 11]
```diff
@@ -0,0 +1,41 @@
...
+     * }
+     * )
+     *
+     * RestView()
+     * @param
+     * @return View
+     *
+     * @throws NotFoundHttpException when page not exist
+     */
+    public function getWorkResourceAction()
```
Comment (spolischook) [inline src/AppBundle/Controller/WorkResourceController.php]: getWorkResourcesAction

File: src/AppBundle/Document/EquipmentResource.php [hunk 12]
```diff
@@ -74,7 +74,7 @@ public static function getReadableQuantityTypes()
     /**
      * @var array
      *
-     * @ODM\ReferenceMany(targetDocument="EquipmentContribute", mappedBy="equipmentResource")
+     * @ODM\ReferenceMany(targetDocument="equipment_contribute", mappedBy="equipmentResource")
```
Comment (spolischook) [inline src/AppBundle/Document/EquipmentResource.php]: ???

File: src/AppBundle/Document/User.php [hunk 13]
```diff
@@ -14,14 +14,12 @@
 class User extends BaseUser //implements DreamUserInterface
 {
     const FAKE_EMAIL_PART = "@example.com";
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:17]: ??

File: src/AppBundle/Document/User.php [hunk 14]
```diff
@@ -14,14 +14,12 @@
...
     /**
      * @var integer
      *
      * @ODM\Id
      */
     protected $id;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:24]: ??

File: src/AppBundle/Document/User.php [hunk 15]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
      * @ODM\Field(type="string")
      */
     protected $firstName;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:32]: ??

File: src/AppBundle/Document/User.php [hunk 16]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
...
     /**
      * @var string
      *
      * @ODM\Field(type="string")
      */
     protected $middleName;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:39]: ??

File: src/AppBundle/Document/User.php [hunk 17]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
...
-//     */
-//    protected $avatar;
-
     /**
      * @var \DateTime
      *
      * @ODM\Field(type="date")
      */
     protected $birthday;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:58]: ??

File: src/AppBundle/Document/User.php [hunk 18]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
...
     /**
      * @var string
      *
      * @ODM\Field(type="string")
      */
     protected $about;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:65]: ??

File: src/AppBundle/Document/User.php [hunk 19]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
...
     /**
      * @var string
      *
      * @ODM\Field(name="vkontakte_id", type="string")
      */
     protected $vkontakteId;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:72]: ??

File: src/AppBundle/Document/User.php [hunk 20]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
...
     /**
      * @var string
      *
      * @ODM\Field(name="facebook_id", type="string")
      */
     protected $facebookId;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:79]: ??

File: src/AppBundle/Document/User.php [hunk 21]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
...
     /**
      * @var string
      *
      * @ODM\Field(name="odnoklassniki_id", type="string")
      */
     protected $odnoklassnikiId;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:86]: ??

File: src/AppBundle/Document/User.php [hunk 22]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
...
-    protected $dreams = array();
-
+    protected $dreams;
     /**
      * @var string
      *
      * @ODM\Field(type="string")
      */
     protected $phone;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:123]: ??

File: src/AppBundle/Document/User.php [hunk 23]
```diff
@@ -29,105 +27,84 @@ class User extends BaseUser //implements DreamUserInterface
...
     /**
      * @var string
      *
      * @ODM\Field(type="string")
      */
     protected $skype;
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:130]: ??

File: src/AppBundle/Document/User.php [hunk 24]
```diff
@@ -137,7 +114,6 @@ public function __construct()
         $this->otherContributions = new \Doctrine\Common\Collections\ArrayCollection();
         $this->dreams = new \Doctrine\Common\Collections\ArrayCollection();
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:140]: ??

File: src/AppBundle/Document/User.php [hunk 25]
```diff
@@ -147,7 +123,6 @@ public function getId()
     {
         return $this->id;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:150]: ??

File: src/AppBundle/Document/User.php [hunk 26]
```diff
@@ -160,7 +135,6 @@ public function setFirstName($firstName)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:163]: ??

File: src/AppBundle/Document/User.php [hunk 27]
```diff
@@ -170,7 +144,6 @@ public function getFirstName()
     {
         return $this->firstName;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:173]: ??

File: src/AppBundle/Document/User.php [hunk 28]
```diff
@@ -183,7 +156,6 @@ public function setMiddleName($middleName)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:186]: ??

File: src/AppBundle/Document/User.php [hunk 29]
```diff
@@ -193,7 +165,6 @@ public function getMiddleName()
     {
         return $this->middleName;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:196]: ??

File: src/AppBundle/Document/User.php [hunk 30]
```diff
@@ -206,7 +177,6 @@ public function setLastName($lastName)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:209]: ??

File: src/AppBundle/Document/User.php [hunk 31]
```diff
@@ -216,7 +186,6 @@ public function getLastName()
     {
         return $this->lastName;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:219]: ??

File: src/AppBundle/Document/User.php [hunk 32]
```diff
@@ -229,7 +198,6 @@ public function setBirthday($birthday)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:232]: ??

File: src/AppBundle/Document/User.php [hunk 33]
```diff
@@ -239,7 +207,6 @@ public function getBirthday()
     {
         return $this->birthday;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:242]: ??

File: src/AppBundle/Document/User.php [hunk 34]
```diff
@@ -252,7 +219,6 @@ public function setAbout($about)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:255]: ??

File: src/AppBundle/Document/User.php [hunk 35]
```diff
@@ -262,7 +228,6 @@ public function getAbout()
     {
         return $this->about;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:265]: ??

File: src/AppBundle/Document/User.php [hunk 36]
```diff
@@ -275,7 +240,6 @@ public function setVkontakteId($vkontakteId)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:278]: ??

File: src/AppBundle/Document/User.php [hunk 37]
```diff
@@ -285,7 +249,6 @@ public function getVkontakteId()
     {
         return $this->vkontakteId;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:288]: ??

File: src/AppBundle/Document/User.php [hunk 38]
```diff
@@ -298,7 +261,6 @@ public function setFacebookId($facebookId)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:301]: ??

File: src/AppBundle/Document/User.php [hunk 39]
```diff
@@ -308,7 +270,6 @@ public function getFacebookId()
     {
         return $this->facebookId;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:311]: ??

File: src/AppBundle/Document/User.php [hunk 40]
```diff
@@ -321,7 +282,6 @@ public function setOdnoklassnikiId($odnoklassnikiId)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:324]: ??

File: src/AppBundle/Document/User.php [hunk 41]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
     {
         return $this->odnoklassnikiId;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:334]: ??

File: src/AppBundle/Document/User.php [hunk 42]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Add favoriteDream
      *
-     * @param AppBundle\Document\Dream $favoriteDream
+     * @param \AppBundle\Document\Dream $favoriteDream
      */
     public function addFavoriteDream(\AppBundle\Document\Dream $favoriteDream)
     {
         $this->favoriteDreams[] = $favoriteDream;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:347]: ??

File: src/AppBundle/Document/User.php [hunk 43]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Remove favoriteDream
      *
-     * @param AppBundle\Document\Dream $favoriteDream
+     * @param \AppBundle\Document\Dream $favoriteDream
      */
     public function removeFavoriteDream(\AppBundle\Document\Dream $favoriteDream)
     {
         $this->favoriteDreams->removeElement($favoriteDream);
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:360]: ??

File: src/AppBundle/Document/User.php [hunk 44]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Get favoriteDreams
      *
-     * @return Doctrine\Common\Collections\Collection $favoriteDreams
+     * @return \Doctrine\Common\Collections\Collection $favoriteDreams
      */
     public function getFavoriteDreams()
     {
         return $this->favoriteDreams;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:370]: ??

File: src/AppBundle/Document/User.php [hunk 45]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Add financialContribution
      *
-     * @param AppBundle\Document\FinancialContribute $financialContribution
+     * @param \AppBundle\Document\FinancialContribute $financialContribution
      */
     public function addFinancialContribution(\AppBundle\Document\FinancialContribute $financialContribution)
     {
         $this->financialContributions[] = $financialContribution;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:383]: ??

File: src/AppBundle/Document/User.php [hunk 46]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Remove financialContribution
      *
-     * @param AppBundle\Document\FinancialContribute $financialContribution
+     * @param \AppBundle\Document\FinancialContribute $financialContribution
      */
     public function removeFinancialContribution(\AppBundle\Document\FinancialContribute $financialContribution)
     {
         $this->financialContributions->removeElement($financialContribution);
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:396]: ??

File: src/AppBundle/Document/User.php [hunk 47]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Get financialContributions
      *
-     * @return Doctrine\Common\Collections\Collection $financialContributions
+     * @return \Doctrine\Common\Collections\Collection $financialContributions
      */
     public function getFinancialContributions()
     {
         return $this->financialContributions;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:406]: ??

File: src/AppBundle/Document/User.php [hunk 48]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Add equipmentContribution
      *
-     * @param AppBundle\Document\EquipmentContribute $equipmentContribution
+     * @param \AppBundle\Document\EquipmentContribute $equipmentContribution
      */
     public function addEquipmentContribution(\AppBundle\Document\EquipmentContribute $equipmentContribution)
     {
         $this->equipmentContributions[] = $equipmentContribution;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:419]: ??

File: src/AppBundle/Document/User.php [hunk 49]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Remove equipmentContribution
      *
-     * @param AppBundle\Document\EquipmentContribute $equipmentContribution
+     * @param \AppBundle\Document\EquipmentContribute $equipmentContribution
      */
     public function removeEquipmentContribution(\AppBundle\Document\EquipmentContribute $equipmentContribution)
     {
         $this->equipmentContributions->removeElement($equipmentContribution);
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:432]: ??

File: src/AppBundle/Document/User.php [hunk 50]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Get equipmentContributions
      *
-     * @return Doctrine\Common\Collections\Collection $equipmentContributions
+     * @return \Doctrine\Common\Collections\Collection $equipmentContributions
      */
     public function getEquipmentContributions()
     {
         return $this->equipmentContributions;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:442]: ??

File: src/AppBundle/Document/User.php [hunk 51]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Add workContribution
      *
-     * @param AppBundle\Document\WorkContribute $workContribution
+     * @param \AppBundle\Document\WorkContribute $workContribution
      */
     public function addWorkContribution(\AppBundle\Document\WorkContribute $workContribution)
     {
         $this->workContributions[] = $workContribution;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:455]: ??

File: src/AppBundle/Document/User.php [hunk 52]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Remove workContribution
      *
-     * @param AppBundle\Document\WorkContribute $workContribution
+     * @param \AppBundle\Document\WorkContribute $workContribution
      */
     public function removeWorkContribution(\AppBundle\Document\WorkContribute $workContribution)
     {
         $this->workContributions->removeElement($workContribution);
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:468]: ??

File: src/AppBundle/Document/User.php [hunk 53]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Get workContributions
      *
-     * @return Doctrine\Common\Collections\Collection $workContributions
+     * @return \Doctrine\Common\Collections\Collection $workContributions
      */
     public function getWorkContributions()
     {
         return $this->workContributions;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:478]: ??

File: src/AppBundle/Document/User.php [hunk 54]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Add otherContribution
      *
-     * @param AppBundle\Document\OtherContribute $otherContribution
+     * @param \AppBundle\Document\OtherContribute $otherContribution
      */
     public function addOtherContribution(\AppBundle\Document\OtherContribute $otherContribution)
     {
         $this->otherContributions[] = $otherContribution;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:491]: ??

File: src/AppBundle/Document/User.php [hunk 55]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Remove otherContribution
      *
-     * @param AppBundle\Document\OtherContribute $otherContribution
+     * @param \AppBundle\Document\OtherContribute $otherContribution
      */
     public function removeOtherContribution(\AppBundle\Document\OtherContribute $otherContribution)
     {
         $this->otherContributions->removeElement($otherContribution);
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:504]: ??

File: src/AppBundle/Document/User.php [hunk 56]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Get otherContributions
      *
-     * @return Doctrine\Common\Collections\Collection $otherContributions
+     * @return \Doctrine\Common\Collections\Collection $otherContributions
      */
     public function getOtherContributions()
     {
         return $this->otherContributions;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:514]: ??

File: src/AppBundle/Document/User.php [hunk 57]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Add dream
      *
-     * @param AppBundle\Document\Dream $dream
+     * @param \AppBundle\Document\Dream $dream
      */
     public function addDream(\AppBundle\Document\Dream $dream)
     {
         $this->dreams[] = $dream;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:527]: ??

File: src/AppBundle/Document/User.php [hunk 58]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Remove dream
      *
-     * @param AppBundle\Document\Dream $dream
+     * @param \AppBundle\Document\Dream $dream
      */
     public function removeDream(\AppBundle\Document\Dream $dream)
     {
         $this->dreams->removeElement($dream);
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:540]: ??

File: src/AppBundle/Document/User.php [hunk 59]
```diff
@@ -331,187 +291,168 @@ public function getOdnoklassnikiId()
...
      * Get dreams
      *
-     * @return Doctrine\Common\Collections\Collection $dreams
+     * @return \Doctrine\Common\Collections\Collection $dreams
      */
     public function getDreams()
     {
         return $this->dreams;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:550]: ??

File: src/AppBundle/Document/User.php [hunk 60]
```diff
@@ -524,7 +465,6 @@ public function setPhone($phone)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:563]: ??

File: src/AppBundle/Document/User.php [hunk 61]
```diff
@@ -534,7 +474,6 @@ public function getPhone()
     {
         return $this->phone;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:573]: ??

File: src/AppBundle/Document/User.php [hunk 62]
```diff
@@ -547,7 +486,6 @@ public function setSkype($skype)
 
         return $this;
     }
-
```
Comment (spolischook) [inline src/AppBundle/Document/User.php:586]: ??

File: src/AppBundle/Tests/Controller/IndexControllerTest.php [hunk 63]
```diff
@@ -1,16 +0,0 @@
-<?php
```
Comment (spolischook) [inline src/AppBundle/Tests/Controller/IndexControllerTest.php:1]: why remove it?

**Comment by spolischook** [comment]
use @View annotation

**Pull request merged**